include/dpp/channel.h

```diff
@@ -123,6 +160,15 @@ class channel : public managed {
 	/** Permission overwrites to apply to base permissions */
 	std::vector<permission_overwrite> permission_overwrites;
 
+	/** Approximate count of messages in a thread (threads) */
+	uint8_t message_count;
+
+	/** Approximate count of members in a thread (threads) */
+	uint8_t member_count;
```

Comment:
i already know these oddball sizes are correct - queried originally in webtax's version :)